### PR TITLE
Add onbuild and slim versions

### PR DESCRIPTION
**Pull request by ruimarinho**
(Includes previous commit under PR #4)

### CODE REVIEW

File: 5/slim/Dockerfile [hunk 1]
```diff
@@ -1,7 +1,7 @@
-FROM mhart/alpine-node:5.5.0
+FROM mhart/alpine-node:5
 
 # Add gcc and git support for native dependencies.
-RUN apk add --update g++ gcc git make python && rm -rf /var/cache/apk/*
+RUN apk --no-cache --virtual buildDependencies add g++ gcc git make python
```
Comment (nunofgs) [inline 5/slim/Dockerfile:4]: Hmm, this made me wonder if we really need python support? Also, should we upgrade to python3 here?

**Comment by nunofgs** [comment]
Can you rebase please?

File: 5/slim/Dockerfile (hunk 1 above)
Comment (ruimarinho) [inline 5/slim/Dockerfile:4]: It is needed for gyp.

**Comment by ruimarinho** [comment]
Rebased.

**Comment by ruimarinho** [comment]
All done @nunofgs.

**Pull request merged**